index.html

```diff
@@ -42,187 +42,231 @@ <h4>I’am a Software Developer</h4>
+    <!-- About Section Start -->
+    <section class="rectangle">
+        <div class="about">
+            <h1 class="about-title">About me</h1>
+            <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="about-btn"><span class="about-btn-title">Get My Resume</span></a>
         </div>
-        <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-        </ul>
-        <div class="skill-frame-1 left-skill">
-            <img class="img-rectangle" src="/images/square.png" alt=""> <span class="img-title">Frameworks</span>
+        <div class="divider">
+            <div class="vector-1"></div>
         </div>
-        <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>Rspec</li>
-            <li>CampyBara</li>
-            <li>Selenium</li>
-        </ul>
-        <div class="skill-frame-1 right-skill">
-            <img class="img-rectangle" src="/images/circle.png" alt=""> <span class="img-title">Skills</span>
+        <!-- Skill div Start -->
+        <div class="skill">
+            <div class="skill-frame-1">
+                <img class="img-rectangle" src="/images/rectangle.png" alt=""> <span class="img-title">Language</span>
+            </div>
+            <ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+            <div class="skill-frame-1 left-skill">
+                <img class="img-rectangle" src="/images/square.png" alt=""> <span class="img-title">Frameworks</span>
+            </div>
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>Rspec</li>
+                <li>CampyBara</li>
+                <li>Selenium</li>
+            </ul>
+            <div class="skill-frame-1 right-skill">
+                <img class="img-rectangle" src="/images/circle.png" alt=""> <span class="img-title">Skills</span>
+            </div>
+            <ul>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
         </div>
-        <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-        </ul>
-    </div>
-    <!-- Skill div End -->
-</section>
-<!-- About Section End -->
+        <!-- Skill div End -->
+    </section>
+    <!-- About Section End -->
+
+    <!-- Contact Form Start -->
+
+
+    <section>
```

Comment:
- [ ] As in the project requirement, the contact form is within the footer section. Therefore, kindly use the `footer` tag instead for this section. For example:
```
<footer>
   <form  action="">
      <input type="text" class="name" maxlength="30">
      <input type="email" class="email" name="email">
   </form>
</footer>
```
![image](https://user-images.githubusercontent.com/44057863/177408449-fb85550c-2077-49e0-b1fa-daebd51975d6.png)